openapi/README.md

```diff
@@ -0,0 +1,6 @@
+# Modifying the API Specification
```

Comment:
this exists in development docs too. maybe we could consolidate (if needed)

Reply:
oh wow, didn't know this existed. Rest of the PR still needs work, will get to it soon!